larq_compute_engine/tflite/tests/bconv2d_test.cc

```diff
@@ -34,6 +35,15 @@
 
 namespace tflite {
 
+constexpr int Padding_ONE = Padding_MAX + 1;
+
+const char* GetPaddingName(enum Padding padding) {
+  if (padding == Padding_VALID) return "VALID";
+  if (padding == Padding_SAME) return "SAME";
+  if (padding == Padding_ONE) return "ONE";
+  return "UNKNOWN";
```

Comment:
What do you think about this?
```suggestion
  switch (padding) {
    case Padding_VALID: return "VALID";
    case Padding_SAME: return "SAME";
    case Padding_ONE: return "ONE";
    default: return "UNKNOWN";
  }
```

(50% of me is just looking for an excuse to try out the new GitHub multi-line suggestions 😉)

Reply:
I applied the change in my own commit instead of the github one, because the linter wanted it on separate lines.